src/Microsoft.Azure.WebJobs.Host/Tables/TableClient.cs

```diff
@@ -110,7 +110,26 @@ public static void ValidateAzureTableName(string tableName)
 
         public static bool IsValidAzureTableName(string tableName)
         {
-            return Regex.IsMatch(tableName, "^[A-Za-z][A-Za-z0-9]{2,62}$");
+            // Table Name Rules are summerized in this page https://msdn.microsoft.com/en-us/library/azure/dd179338.aspx
+            // Table names may contain only alphanumeric characters.
+            if (!Regex.IsMatch(tableName, "^[A-Za-z][A-Za-z0-9]{2,62}$"))
```

Comment:
actually you are right I misinterpreted it, reverting that change and will only keep the comment referencing the naming rules